x11/homebank/Portfile

```diff
@@ -20,9 +21,13 @@ long_description    HomeBank is the free software you have always wanted to \
 homepage            http://homebank.free.fr/
 master_sites        ${homepage}public/
 
-checksums           rmd160  9279a042bbcf506b2bedd9809d60c22116bb9462 \
-                    sha256  acc8221fffbbe4b6ddc23fe845fa061cae2ecf7643f2ed858adc9a2e6e2295a5 \
-                    size    2601784
+checksums           rmd160  57eef07611d930a80d1d72672a014323c6ca3215 \
+                    sha256  79a89ab8816a5973fa6afe75157fa375953795c79c224d510e8af0afed2512d2 \
+                    size    2673200
+
+# Needed for intltool
+configure.perl  ${prefix}/bin/perl5.28
+configure.args  INTLTOOL_PERL=${configure.perl}
```

Comment:
I'm not seeing a build dependency on perl5.28. Please add that.

Reply:
In pull request #4109, intltool has a dependency on p5.28-xml-parser.
https://github.com/macports/macports-ports/pull/4109#issuecomment-485194850

Reply:
Please autoreconf instead of doing this. There's no reason for a client of intltool to care which version of perl intltool uses internally. See my explanation in #4109.